mantidproject/mantid
# Improve DirectILLDiagnostics reporting, default reduction options for IN4 and IN6

This PR removes the user masked detectors part of the *OutputReport* string of `DirectILLDiagnostics`. Additionally, the automatically picked parameters are now phrased explicitly in the IPFs of IN4 and IN6 so users don't need to cross check the used values from documentation/developers.

**To test:**

It should be pretty simple to see from the code changes that the extra reporting is gone. However, if in doubt, make sure Mantid unit test data dir is in data search directories. Then run:
```python
DirectILLCollectData(Run='ILL/IN4/084446.nxs', OutputWorkspace='out', OutputEPPWorkspace='epps')
result = DirectILLDiagnostics('out', EPPWorkspace='epps', MaskedDetectors=[1,2,3], MaskedComponents='rosace')
print(result.OutputReport)
```

The printed output should not mention user masked detectors.

Fixes #21469.

**Release Notes** 

Direct inelastic release notes were updated.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
